pkg/vadmin/create_db_at.go

```diff
@@ -76,3 +77,23 @@ func (a *Admintools) genCreateDBCmd(s *createdb.Parms) []string {
 	}
 	return cmd
 }
+
+// GenCmd will return the command line options for calling admintools -t create_db.
+func (d *DBCreator) GenCmd() []string {
+	return d.Admintools.genCreateDBCmd(&d.Parms)
+}
+
+// GetInitiator returns the initiator pod name.
+func (d *DBCreator) GetInitiator() types.NamespacedName {
+	return d.Parms.Initiator
+}
+
+// LogFailure will log and record an event for an admintools -t create_db failure
+func (d *DBCreator) LogFailure(stdout string, err error) (ctrl.Result, error) {
+	return d.Admintools.logFailure("create_db", events.ReviveDBFailed, stdout, err)
```

Comment:
We are using the wrong event here:
```suggestion
	return d.Admintools.logFailure("create_db", events.CreateDBFailed, stdout, err)
```